pkg/didcomm/transport/http/inbound.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package http
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// InboundDIDCommHandler struct to pass details for request handlers
+type InboundDIDCommHandler struct {
+	Path        string
+	HandlerFunc http.HandlerFunc
+}
+
+// NewInboundHandler will create a new handler to enforce Did-Comm HTTP transport specs
+// then routes processing for the passed 'inURL' and 'handlerFunc' arguments.
+// Both 'inURL' and 'handlerFunc' are mandatory for inbound communications.
```

Comment:
It would help to provide a description of these arguments - esp in the constructor.

Reply:
'inURL' depicts the inbound http path in the POST request
'handlerFunc' is the HTTP handlerFunc that handles the inbound request

I can add these in the comments